app/models/service_template.rb

```diff
@@ -134,7 +134,10 @@ def create_service(service_task, parent_svc = nil)
       svc.add_resource(sr.resource, nh) unless sr.resource.nil?
     end
 
-    parent_svc.add_resource!(svc) unless parent_svc.nil?
+    if parent_svc
```

Comment:
@tinaafitz Please review

Found this issue while working with @syncrou.  When the child services were attached to the parent bundle the metadata in the `service_resource` join table was not being carried along.  This caused an issue when we tried to identify what start/stop actions were defined in the service template.

Now instead of attaching the service to the bundle with an empty `service_resource` join we are copying the properties of the from the service_template when we create the join between the bundle and the child service.